### PR TITLE
Update career progression content

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/DiY2gM75/6986

### CODE REVIEW

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md [hunk 1]
```diff
@@ -48,81 +55,74 @@ keywords:
...
+You may also receive additional payments on top of your regular salary for taking on extra responsibilities. These are known as teaching and learning responsibility (TLR) payments. 
 
-## How long does it take to go from an early career teacher to a headteacher?
+This could be a one-off payment for a specific project, for example, creating a timetable. Or it might be an extra responsibility attached to your job role, for example, being the head of department. 
 
-<strong>Sian:</strong> I think 8 to 10 years is realistic to become a headteacher for primary schools although you may be able to climb the career ladder a bit quicker.
+Depending on your responsibilities, you could earn up to an extra $salaries_tlr_max$. Your school will determine how much you receive.
 
-<strong>Elaine:</strong> If you’re driven you could probably get to headship in between 9 to 12 years. It took me longer but at the time it suited me to stay where I was.
+## National professional qualifications
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md:122]: Should NPQs come before salary? Feels like they are linked to the section before?

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md [hunk 2]
```diff
@@ -48,81 +55,74 @@ keywords:
...
 
-<strong>Chris:</strong> If you want to develop your leadership skills there are national professional qualification (NPQ) leadership courses available to help teachers progress their careers.
+National professional qualifications (NPQs) are professional development courses for teachers and leaders to develop skills, progress careers and improve school outcomes.  
 
-## How has teaching helped your professional development?
+There are 2 types of NPQs: specialist and leadership. Specialist NPQs can help you develop your classroom teaching or specialism. Leadership NPQs can help you develop your leadership skills or to progress to a leadership role.
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md:126]: Not sure if it is worth mentioning funding - there is a small para at the end of [the GOV.UK page](https://www.gov.uk/guidance/national-professional-qualification-npq-courses) on this (but that might be too much detail)

Would be good to review this with the CPD team

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md [hunk 3]
```diff
@@ -1,16 +1,24 @@
 ---
-title: "How to move up the career ladder in teaching"
-heading: "How to move up the career ladder in teaching"
+title: "Career progression in teaching"
+heading: "Career progression in teaching"
 description: |-
-  Teacher training advisers talk about teacher development and how you can progress a career in teaching.
+  Find out about all the opportunities for progression as a teacher, including early career support, salary progression, and professional qualifications.
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md:5]:
```suggestion
  Find out about the opportunities for progression as a teacher, including salary progression, professional qualifications, and mentorship.
```

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md [hunk 4]
```diff
@@ -48,81 +55,75 @@ keywords:
...
+[Find out more about early career teachers](/life-as-a-teacher/teaching-as-a-career/early-career-teachers).
 
-## How do you know when you’re ready to apply for a teaching promotion?
+## Taking on more responsibility
 
-<strong>Fran:</strong> When I was working as an English teacher I got offered a fixed-term teaching and learning responsibility (TLR) payment to work as an assistant head of year. It was a good way for me to test the water to see if I could deal with the demands of a more senior role. Fixed-term TLRs are usually for one-off projects. There are also TLRs that are for permanent roles. You apply for these as you would any job and get paid the TLR while you're doing it.
+Whether it’s during your time as an early career teacher, or at other stages in your career, you can take on more responsibility to help progress your career.
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md:81]:
```suggestion
Whether it’s during your time as an early career teacher, or at other stages in your career, you can take on more responsibility to help you progress and develop.
```

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md [hunk 5]
```diff
@@ -48,81 +55,75 @@ keywords:
...
+* headteacher 
 
-<strong>Fran:</strong> Realistically you’d want to do at least a year or two in the same setting before then thinking about applying for a role with more responsibility. But you have to prove your worth to get a teaching and learning responsibility (TLR) payment.
+These roles tend to combine elements of pastoral care, academic and curriculum management, and staff development.  
 
-The headteacher at my school was looking to create an alternative for key stage 3 (KS3) pupils who weren’t able to access the curriculum properly. As part of a shift in my lead practitioner responsibility for English I was given a permanent TLR to research and implement this. I had to come up with case studies and a proposal. It's about showing you’re prepared to invest in your personal development and know what’s going on out there. You have to keep up with legislation, news and topical priorities.
+$gloria$
 
-<strong>Sian:</strong> It may be the case that your school hasn’t got the budget for a TLR. So if you really want to take on more responsibility you may need to think about moving school.
+### Become a mentor
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md:103]:
```suggestion
### Mentor roles
```

File: app/views/content/shared/quotes/_quote_progression_aaron.html.erb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<%= render Content::QuoteComponent.new(
+  text: "The continued professional development I've received throughout my career so far has shaped me.",
+  name: "Aaron, science teacher",
+  large: false
```
Comment (gemmadallmandfe) [inline app/views/content/shared/quotes/_quote_progression_aaron.html.erb:4]: False is the default for the quote component, so you should be able to remove this line

File: app/views/content/shared/quotes/_quote_progression_gloria.html.erb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<%= render Content::QuoteComponent.new(
+  text: "I started out as a regular classroom teacher, but over the years, I’ve grown and taken on different roles. I became head of year to understand how to support students beyond academics, and later, I moved into a leadership role in teaching maths. Every step of the way, I’ve learned so much about how students learn, and I’m still learning every day.",
+  name: "Gloria, maths lead practitioner",
+  large: false
```
Comment (gemmadallmandfe) [inline app/views/content/shared/quotes/_quote_progression_gloria.html.erb:4]: False is the default for the quote component, so you should be able to remove this line

File: app/views/content/shared/quotes/_quote_progression_aaron.html.erb [hunk 8]
```diff
@@ -0,0 +1,4 @@
+<%= render Content::QuoteComponent.new(
+  text: "The continued professional development I've received throughout my career so far has shaped me. It's equipped me with a range of skills that enable me to be a better version of myself and subsequently lead to career progression.",
```
Comment (gemmadallmandfe) [inline app/views/content/shared/quotes/_quote_progression_aaron.html.erb:2]:
```suggestion
  text: "The continued professional development I've received throughout my career so far has shaped me. It's equipped me with a range of skills that enable me to be a better version of myself and has subsequently led to career progression.",
```

I find this end of this quote slightly hard to read - does it need tweaking slightly? But I may have misunderstood the context

File: app/views/content/shared/quotes/_quote-teaching-abigail.html.erb [hunk 9]
```diff
@@ -0,0 +1,12 @@
+<div class="row">
+  <div class="col-space-l col-space-l-top">
+    <%= render Content::QuoteWithImageComponent.new(
+      title: "Teacher career progresion stories",
```
Comment (gemmadallmandfe) [inline app/views/content/shared/quotes/_quote-teaching-abigail.html.erb:4]:
```suggestion
      title: "Teacher career progression stories",
```

File: app/views/content/shared/quotes/_quote-teaching-abigail.html.erb [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+      text: "<p>I completed the National Professional Qualification for Senior Leadership (NPQSL), which has involved running a project across the whole school. This has given me the opportunity to work more closely with the senior leadership team and has helped strengthen my relationships with colleagues in other departments in the school.</p><a href='/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story'>Read Abigail's career progression story</a>",
```
Comment (gemmadallmandfe) [inline app/views/content/shared/quotes/_quote-teaching-abigail.html.erb:5]:
```suggestion
      text: "<p>I completed the National Professional Qualification for Senior Leadership (NPQSL), which involved running a project across the whole school. This has given me the opportunity to work more closely with the senior leadership team and has helped strengthen my relationships with colleagues in other departments in the school.</p><a href='/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story'>Read Abigail's career progression story</a>",
```
Suggested change (ignore as needed) plus do we need a full-stop after the link text?

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression/_article.html.erb [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+    <h2 class="heading--box-blue">Early career progression</h2>
+    <p>The thought of teaching your own class after you’ve completed your teacher training may feel a bit daunting. But teaching offers the opportunity for continuous professional development so you’re always learning new skills.</p>
+    <p>If your course leads to qualified teacher status (QTS), you’ll receive 2 years of further support once you’ve completed your training. You’ll be known as an early career teacher (ECT) during this time and receive training and support based on the early career framework (ECF).</p>
+
+    <p><a href="/life-as-a-teacher/teaching-as-a-career/early-career-teachers">Find out more about early career teachers</a>.</p>
+
+    <h2 class="heading--box-blue">Taking on more responsibility</h2>
+    <p>Whether it’s during your time as an ECT, or at other stages in your career, you can take on more responsibility to help you progress and develop.</p>
+
+    <p>There's also no set path for how and when you can take on a more responsibility. This gives you the opportunity to progress at your own pace.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression/_article.html.erb:16]:
```suggestion
    <p>There's also no set path for how and when you can take on more responsibility. This gives you the opportunity to progress at your own pace.</p>
```

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression/_article.html.erb [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+    <p>You can grow your responsibilites by taking on new roles, such as:</p>
+    <ul>
+      <li>form tutor</li>
+      <li>subject leader</li>
+      <li>special educational needs co-ordinators (SENCO)</li>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression/_article.html.erb:22]:
```suggestion
      <li>special educational needs co-ordinator (SENCO)</li>
```

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression/_article.html.erb [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+    <p><%= render 'content/shared/quotes/quote_progression_gloria' %></p>
+
+    <h3>Mentor roles</p>
+    <p>You can become a mentor for ECTs. In this role, you’ll receive training on mentorship and coach a newly qualified teacher at the start of their career.</p>
+
+    <p>You can also mentor trainee teachers during their initial teacher training school placements.</p>
+
+    <h2 class="heading--box-blue">National professional qualifications</h2>
+    <p>National professional qualifications (NPQs) are designed to provide training and support for teachers and leaders at all levels. They've been designed with education professionals in mind, and can be completed flexibly around your teaching commitments.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression/_article.html.erb:39]: Slight repetition re `designed` - worth tweaking?

Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression/_article.html.erb:39]:
```suggestion
    <p>National professional qualifications (NPQs) provide training and support for teachers and leaders at all levels. They've been designed with education professionals in mind, and can be completed flexibly around your teaching commitments.</p>
```

File: app/views/content/shared/quotes/_quote-teaching-abigail.html.erb (hunk 10 above)
Comment (emmaachesongray) [reply app/views/content/shared/quotes/_quote-teaching-abigail.html.erb:5]:
```suggestion
      text: "<p>I completed the National Professional Qualification for Senior Leadership (NPQSL), which involved running a project across the whole school. This has given me the opportunity to work more closely with the senior leadership team and has helped strengthen my relationships with colleagues in other departments in the school.</p><p><a href='/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story'>Read Abigail's career progression story</a>".</p>,
```

File: app/views/content/shared/quotes/_quote_progression_aaron.html.erb (hunk 8 above)
Comment (emmaachesongray) [reply app/views/content/shared/quotes/_quote_progression_aaron.html.erb:2]:
```suggestion
  text: "The continued professional development I've received throughout my career so far has shaped me. It's equipped me with a range of skills that enabled me to be a better version of myself and subsequently led to career progression.",
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

My only thought is if we want to add an adviser promo at the end?

**Comment by emmaachesongray** [comment]
Approved by policy

**Pull request merged**